openage/convert/tool/singlefile.py

```diff
@@ -42,11 +42,12 @@ def main(args, error):
     file_path = Path(args.filename)
     file_extension = file_path.suffix[1:].lower()
 
-    if not args.palettes_path:
-        raise Exception("palettes-path needs to be specified")
+    if args.mode not in ["drs-wav", "wav"] or file_extension != "wav":
```

Comment:
This condition will still be `True` if no `args.mode` was given, but the file extension is `wav`.

Better use `not (args.mode in ["drs-wav", "wav"] or file_extension == "wav")`

Reply:
ok yeah

Reply:
Boolean logic :D